src/shared/UI/StarRate/index.tsx

```diff
@@ -0,0 +1,15 @@
+import { StarIcon } from "@shared";
+
+interface StarRateProps {
+  rate: number;
+}
+
+export default function StarRate ({ rate }: StarRateProps): JSX.Element {
+  return (
+    <div className="flex w-5/12 justify-between">
```

Comment:
Please try to use semantic html. For example i found this example: https://codepen.io/nordskill/pen/eYLpgqq.
Just play with it - maybe u will find better way.

Reply:
I think that using <meter/> is not a good idea. I created a StarRate element for reusing it in the future to set rates for products. If we use <meter/> this feature will not work. Also, this tag must be overstyled to get the same result in different browsers.

Reply:
I thing it is not very difficult - my implementation: https://codesandbox.io/p/sandbox/quirky-sanderson-5xrrqj?file=/src/main.tsx:10,1